tests/acceptance/filetypes.robot

```diff
@@ -0,0 +1,104 @@
+*** Settings ***
+Documentation   Check if rflint reads supported file formats
+Library         OperatingSystem
+Library         Process
+Library         String
+Resource        SharedKeywords.robot
+
+
+*** Test Cases ***
+All Supported File Types
+    [Template]      Run Rflint And Verify There Are No Errors For Supported File Types
+    robot   resource    tsv
+
+.robot And .resource Supported File Types
+    [Template]      Run Rflint And Verify There Are No Errors For Supported File Types
+    robot   resource
+
+.resource And .tsv Supported File Types
+    [Template]      Run Rflint And Verify There Are No Errors For Supported File Types
+            resource    tsv
```

Comment:
Misaligned

Reply:
I did it on purpose, please look at the whole file. But I can revert it if it looks strange. What do you think?

Reply:
Oh I see, thanks to this format it's easy to see that you're testing this or that file format combination. But maybe you could use this approach?
`
*** Test Cases ***
Test Supported File Types
[Template]      Run Rflint And Verify There Are No Errors For Supported File Types
robot   resource    tsv
robot   resource
           resource    tsv
robot                    tsv
robot
           resource
`

Reply:
Yeah, another approach is to use arguments embedded in keyword name. I will think which one is the best for this use.